talaiot/src/test/kotlin/com/cdsap/talaiot/metrics/DefaultBuildDataProviderTest.kt

```diff
@@ -36,9 +36,9 @@ class DefaultBuildMetricsProviderTest : BehaviorSpec({
                     "buildId" to "12",
                     "buildInvocationId" to "123",
                     "osVersion" to "Linux 1.4",
-                    "maxWorkers" to 2,
+                    "maxWorkers" to 2.toInt(),
                     "javaRuntime" to "1.2",
-                    "cpuCount" to 4,
+                    "cpuCount" to 4.toInt(),
```

Comment:
but I don't know why it's required now :(

Reply:
Checking too

Reply:
tested even changing the order like:
```
 mapOf(
                "maxWorkers" to 2,
                "cpuCount" to 4,
                 ...

```
And getting the same error.  I tried with Kotlin 1.3.72 and  same result. I would like to know how is possible the cast for the map is not working as expected(and was working before).

Merging the PR, is not blocker.